netbox/dcim/api/views.py

```diff
@@ -501,7 +501,7 @@ def napalm(self, request, pk):
                 response[method] = {'error': 'Only get_* NAPALM methods are supported'}
                 continue
             try:
-                response[method] = decode_dict(getattr(d, method)())
+                response[method] = decode_dict(getattr(d, method)(), decode_keys=False)
```

Comment:
Can you try removing the call to `decode_dict()` entirely and test again? Per @thatmattlove's comments I think we can just remove it. Any drivers which don't properly format their output will need to be fixed upstream.

```suggestion
                response[method] = getattr(d, method)()
```

Reply:
Yes, it is working without `decode_dict`. I also agree with the fact that format output should be handled directly by napalm.